packages/amplify-graphql-auth-transformer/src/utils/schema.ts

```diff
@@ -34,7 +35,7 @@ export const fieldIsList = (fields: ReadonlyArray<FieldDefinitionNode>, fieldNam
 
 export const getModelConfig = (directive: DirectiveNode, typeName: string, isDataStoreEnabled = false): ModelDirectiveConfiguration => {
   const directiveWrapped: DirectiveWrapper = new DirectiveWrapper(directive);
-  const options = directiveWrapped.getArguments<ModelDirectiveConfiguration>({
+  const options = directiveWrapped.getArguments({
```

Comment:
Is there a reason we need to remove this type?

Reply:
it doesn't, updated